doc/Makefile

```
@@ -78,3 +78,6 @@ linkcheck:
 
 linkcheck-grep:
 	@! grep -h "^.*:.*: \[\(\(local\)\|\(broken\)\)\]" _build/linkcheck/output.txt
+
+serve-html:
+	python -m http.server --directory "$(BUILDDIR)"/html
```

`python -m http.server` always uses port 8000. I often am doing local development on that port and it would be great to have this auto-select an available port.

```suggestion
	python -m http.server 0 --directory "$(BUILDDIR)"/html
```